docs/tools/node_maintain.md

```diff
@@ -2,45 +2,60 @@
 
 ## Overview
 
-Graceful decommissioning could minimize the impact on existing job when removing unhealthy nodes. 
-This is a tool to help you achieve it in OpenPAI. 
+This [tool](../../src/tools/node_maintain.py) help you to gracefully decommission unhealthy nodes in OpenPAI. 
 
-In OpenPAI, an unhealthy node-list records the nodes to be decommissioned. 
-Components could sync the list and decommission them as needed.
-
-**NOTES: For now, only YARN nodes are decommissioned, HDFS doesn't.**
+**NOTES: This tool should be invoked in the dev-box, under `{PAI_ROOT_DIR}/src/tools`. For now, it only supports decommissioning YARN nodes.**
 
 ## Commands
 
-Currently, node-list update and component refresh are both manual actions. 
-This tool provides corresponding command for them. 
-The only mandatory argument for all commands is `master_ip`. Usually, components share the same master ip,
-in different case(i.e. load-balance), you could overwrite `master_ip` separately by `--api-server-ip` and `--resource-manager-ip`.
-`api-server-ip` should be the kubernetes load-balance ip in [kubernetes-configuration](../../examples/cluster-configuration/kubernetes-configuration.yaml).
-`resource-manager-ip` should be the pai-master node ip of machine-list in [layout](../../examples/cluster-configuration/layout.yaml)
+This tool provides commands to query alerting nodes from prometheus and operate blacklist.
+A common invoking looks like:
+```bash
+python node_maintain.py {object} {action} {arguments}
+```
+
+### Get current unhealthy GPU nodes
 
-So, a common invoking looks like:
+###### 1. Query prometheus by this tool
 ```bash
-python node_maintain.py {master_ip} sub-command sub-command-arguments
+python node_maintain.py badgpus get -m {prometheus_ip} [--prometheus-port {prometheus-port}]
```

Comment:
We'd better add some background about the why we 'query prometheus'. For example: We monitor some gpu metrics using prometheus, we can query the bad GPUs from prometheus.

Reply:
Added